joint_state_broadcaster/joint_state_plugin.xml

```diff
@@ -0,0 +1,7 @@
+<library path="joint_state_broadcaster">
+  <class name="joint_state_broadcaster/JointStateBroadcaster" type="joint_state_broadcaster::JointStateBroadcaster" base_class_type="controller_interface::ControllerInterface">
+  <description>
+    The joint state broadcaster publishes the current joint state
```

Comment:
```suggestion
    The joint state broadcaster publishes the current states from ros2_control system.
```

I propose something more general. Especially because we can publish now all kind of states in the /dynamic_joint_state.